package.json

```diff
@@ -48,6 +48,56 @@
     "appId": "org.ssbc.ssb-first-aid-kit",
     "directories": {
       "output": "dist/installers"
+    },
+    "win": {
+      "target": [
+        {
+          "target": "nsis",
+          "arch": [
+            "x64",
+            "ia32"
+          ]
+        },
+        {
+          "target": "zip",
+          "arch": [
+            "x64",
+            "ia32",
+            "arm64"
+          ]
+        }
+      ]
+    },
+    "mac": {
+      "category": "public.app-category.social-networking"
+    },
+    "dmg": {
+      "iconSize": 160,
+      "contents": [
+        {
+          "x": 180,
+          "y": 170
+        },
+        {
+          "x": 480,
+          "y": 170,
+          "type": "link",
+          "path": "/Applications"
+        }
+      ]
+    },
+    "linux": {
+      "target": [
+        {
+          "target": "AppImage",
+          "arch": [
+            "x64",
+            "ia32",
+            "arm64"
+          ]
+        }
+      ],
+      "category": "Network;Chat"
```

Comment:
chat? :P

Reply:
well gossiping is kinda chatty... :-)

Reply:
well technically this app doesn't do the gossiping, it's more of a gossip-aide :P

Reply:
Ah I saw you changed it now